NEWS

```diff
@@ -1,3 +1,52 @@
+KLEE 1.4.0, 20 July 2017
+========================
+(Incorporating changes from 4 November 2016 up to and including 20 July 2017)
+
```

Comment:
Thanks for creating that. I think we should add a link to the documentation for the release.